code-of-conduct.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
See: https://github.com/ampproject/worker-dom/blob/master/CODE_OF_CONDUCT.md

Can just point to AMP's Code of Conduct.